packages/core/data-transfer/src/strapi/providers/local-source/assets.ts

```diff
@@ -76,6 +100,7 @@ export const createAssetsStream = (strapi: LoadedStrapi): Duplex => {
 
     for await (const file of stream) {
       const isLocalProvider = file.provider === 'local';
+      await signUrl(isLocalProvider, file);
```

Comment:
It might more sense to move the isLocalProvider conditional around this instead of inside it